packages/devel/cmake/package.mk

```diff
@@ -3,11 +3,12 @@
 # Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="cmake"
-PKG_VERSION="3.23.2"
-PKG_SHA256="f316b40053466f9a416adf981efda41b160ca859e97f6a484b447ea299ff26aa"
+PKG_VERSION="3.24.0"
+PKG_SHA256="c2b61f7cdecb1576cad25f918a8f42b8685d88a832fd4b62b9e0fa32e915a658"
 PKG_LICENSE="BSD"
 PKG_SITE="https://cmake.org/"
 PKG_URL="https://cmake.org/files/v$(get_pkg_version_maj_min)/cmake-${PKG_VERSION}.tar.gz"
+PKG_URL="https://cmake.org/files/v3.24/cmake-${PKG_VERSION}.tar.gz"
```

Comment:
wrong url :)

Reply:
> wrong url :)

Thanks for spotting that. (Joys of tracking RCs and squashing.)